tests/wrapper/test_simulation_object_updating_component.py

```diff
@@ -41,7 +41,7 @@ def component(self, traffic_update):
     def configured_component(self, traffic_update):
         # pylint: disable=unused-argument
         component = SimulationObjectUpdatingComponent(
-            sumo_configuration="sumo-config/example.scenario.sumocfg"
+            sumo_configuration="data/sumo/example/sumo-config/example.scenario.sumocfg"
```

Comment:
I recommend you to use path.join for this 😇 It differentiates between OS if you have to use `/` or `\`.